modules/web/themes/halo/samples/components/icons/icons.scss

```diff
@@ -13,7 +21,16 @@
     height: inherit;
   }
 
-  .v-slot-sampler-scroll .v-slot-c-scrollbox {
+  .v-slot-sampler-scroll {
     height: inherit;
   }
+
+  .vbox-icon:hover {
+    background-color: #a3a3a3;
```

Comment:
I suggest using `$v-focus-color` as a background-color and computed `color`, for instance:

```
.vbox-icon:hover {
  background-color: $v-focus-color;

  $font-color: valo-font-color($v-selection-color, 0.9);
  color: $font-color;

  .icon-label > span {
    color: $font-color;
  }
}
```